.github/workflows/notify-on-review-wanted.yml

```diff
@@ -0,0 +1,40 @@
+name: Notify on Review Wanted
+on:
+  issues:
+    types: [labeled]
```

Comment:
It doesn't make sense to trigger that on issues.
```suggestion
```
Although, shouldn't we instead add a job to `.github/workflows/comment-labeled.yml` instead of creating a new workflow?